benefits/remote_working.md

```diff
@@ -1,19 +1,26 @@
 
-Maybe you find that you're most productive WFH during early mornings? Why not start at 6am and finish at 2pm? What if you need to be somewhere at 4pm on a Tuesday? Re-jig your day so you can do this.
+As a highly collaborative agile team we recognise there are tradeoffs when choosing between working from the office and from home. Below we've highlighted some general ettiquette along with guidelines for working remotely on a regular basis.
 
-All we ask is that you:
+## General ettiquette
 
-* Check and clear it with the rest of your team before the end of the previous working day.
+- Check and clear it with the rest of your team before the end of the previous working day
+- Once agree with your team and customers add an event in the remote working calendar
+- Check that you're not needed on-site by a customer
+- Ensure you're available for a daily standup with your team
+- Be a good slack citizen and check in with your team throughout your working day if possible
+- Ensure you're completely clear on your specific workload, down to the task level, for the day or days you wish to work from home, and ensure that you're certain you won't benefit from working closely with others on this
+- Ensure you're not scheduled for any meetings or other sessions
+- Barring any extraordinary circumstances, we like people to be in the office on Friday for [lunch (and hopefully drinks)](/benefits/friday_lunch_drinks.md)
 
-* Check that you're not needed on-site by a customer.
+## Regularly working from home
 
-* Ensure you're available for a daily standup with your team.
+There are additional considerations when regularly working remotely.
 
-* Ensure you're completely clear on your specific workload, down to the task level, for the following day, and that you're certain you won't benefit from working closely with others on this.
-
-* Ensure you're not scheduled for any meetings or other sessions.
-
-Barring any extraordinary circumstances, we like people to be in the office on Friday for [lunch (and hopefully drinks)](/benefits/friday_lunch_drinks.md).
+- Talk to your team and customers ahead of committing yourself to working remotely on a regular basis
+- Once agreed with your team and customers, put a recurring event in the remote working calendar
+- We recommend working from home a maximum of 2 days per week when part of a delivery team – it's important to ensure you get enough face time with your team and customers for asking questions, raising blockers, and showcasing work
+- Again, barring any extraordinary circumstances, we like people to be in the office on Friday
+- Make sure you choose days that will have the least impact on customers – when working with larger organisations their stakeholders time will be less flexible and thus you'll need to ensure there are clearly set expectations around when showcases will be, etc.
```

Comment:
Think this important but I think think the concrete example feels like the only context that impact can occur